Automattic/woocommerce-services
# Fix log file retrieval on Status page to work with new WC 3.4 naming conventions

Fix log file retrieval on Status page to work with new WC 3.4 format that includes the date.

https://github.com/woocommerce/woocommerce/commit/e9b0c6deec21c7befa7805e40c84cebc451d83a8 introduced a date string `YYYY-MM-DD` into the log file names.

To test:
* Verify that an "old style" log still loads on the WCS status page (name like `wc-services-c4fa6ec2cfff7f708e89c42ac4b5a28c.log`)
* Delete that log file
* Perform an action that adds entries to the file, like rates retrieval/taxes
* Verify that entries are displayed on the Status page
